Rakefile

```diff
@@ -87,25 +92,20 @@ namespace :cucumber do
   desc "Run Cucumber features with Safari under each driver"
   task :safari => %w[ capybara:safari selenium:safari watir:safari ]
 
-  desc "Run Cucumber features with PhantomJS under each driver"
-  task :phantomjs => %w[ capybara:poltergeist capybara:phantomjs selenium:phantomjs watir:phantomjs ]
 
   Cucumber::Rake::Task.new 'ci', 'Run Cucumber features and save results in junit xml' do |t| t.cucumber_opts = "-p capybara -p chrome --format junit --out results/cucumber/" end
 
-  Cucumber::Rake::Task.new 'capybara:webkit',       'Run Cucumber features with Capybara driving Webkit'      do |t| t.cucumber_opts = "-p capybara -p webkit" end
-  Cucumber::Rake::Task.new 'capybara:poltergeist',  'Run Cucumber features with Capybara driving Poltergeist' do |t| t.cucumber_opts = "-p capybara -p poltergeist" end
-  Cucumber::Rake::Task.new 'capybara:firefox',      'Run Cucumber features with Capybara driving Firefox'     do |t| t.cucumber_opts = "-p capybara -p firefox" end
-  Cucumber::Rake::Task.new 'capybara:chrome',       'Run Cucumber features with Capybara driving Chrome'      do |t| t.cucumber_opts = "-p capybara -p chrome" end
-  Cucumber::Rake::Task.new 'capybara:phantomjs',    'Run Cucumber features with Capybara driving PhantomJS'   do |t| t.cucumber_opts = "-p capybara -p phantomjs" end
-  Cucumber::Rake::Task.new 'capybara:safari',       'Run Cucumber features with Capybara driving Safari'      do |t| t.cucumber_opts = "-p capybara -p safari" end
+  Cucumber::Rake::Task.new 'capybara:webkit',         'Run Cucumber features with Capybara driving Webkit'            do |t| t.cucumber_opts = "-p capybara -p webkit" end
```

Comment:
I _knew_ I'd regret not spending the time to metaprogram these combinations out 😜